env/work/srw_python/SRWLIB_Example16.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
```

Comment:
SRWLIB_Example16.py should be modified to implement saving some of (or all) result to files (into the folder data_example_16), so that even if Matplotlib, and / or SpiPy is not installed, the example would still be runnable and the main results readable (excluding analytical calculations if SciPy is not installed), as this is the case for all examples SRWLIB_Example01.py - SRWLIB_Example14.py.